aptos-move/framework/aptos-framework/sources/account_v2.move

```diff
@@ -0,0 +1,205 @@
+module aptos_framework::account_v2 {
```

Comment:
can we avoid naming libraries/modules with version numbers in them? I can't recall a library in any programming language called "v2" (though happy to be surprised)